auth/authorizer.go

```diff
@@ -0,0 +1,81 @@
+package auth
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/casbin/casbin"
+	"github.com/dmarro89/dare-db/logger"
+)
+
+const GUEST_USER = "guest"
+const GUEST_ROLE = "guest"
+const DEFAULT_USER = "admin"
+const DEFAULT_ROLE = "admin"
```

Comment:
The default password should be used from `config.toml` file

Reply:
Ok, here just the default role and user.